### DISCUSSION TITLE
Best photo size?

### DISCUSSION THREAD

**Discussion by tvone** [SWHL/RapidVideOCR#65] [Q&A]
When using VSF, how much distance is needed from the position of the letter? Crop the image as close to the text as possible, right?

**Comment by SWHL** [comment]
Yes, just try to get as close to the text as possible

**Reply by tvone** [reply]
![image](https://github.com/user-attachments/assets/0215df50-176e-4a27-a314-a3cc15416f44)
Do I need to adjust this setting to better detect RGBImages files? For example, text color and text border

**Reply by SWHL** [reply]
You can try it with default setting firstly.
